src/services/utils.ts

```diff
@@ -3,18 +3,17 @@
  *
  * @author Katia Aresti
  */
-import { SelectOptionObject } from '@patternfly/react-core/src/components/Select/SelectOption';
 import { KeycloakService } from './keycloakService';
 
 export enum KeyContentType {
-  OctetStream = 'application/octet-stream',
-  OctetStreamHex = 'application/octet-stream; encoding=hex',
   StringContentType = 'java.lang.String',
   DoubleContentType = 'java.lang.Double',
   IntegerContentType = 'java.lang.Integer',
   LongContentType = 'java.lang.Long',
   BooleanContentType = 'java.lang.Boolean',
-  BytesType = 'Bytes'
+  BytesType = 'Bytes',
+  OctetStream = 'application/octet-stream',
```

Comment:
A less 'technical' way of present this is ```byte[] as Base64```

Reply:
done